assimilation_code/modules/observations/DEFAULT_obs_kind_mod.F90

```diff
@@ -489,10 +489,14 @@ module obs_kind_mod
   QTY_ION_O_MIXING_RATIO            = 365, &
   QTY_ATOMIC_H_MIXING_RATIO         = 366
 
+!!MYY: absolute humidity
```

Comment:
I know lots of people habitually put their initials next to the changes, but this is completely unneccesary.
'git blame <filename>'
makes it perfectly clear who modifed what and when.